lib/config.js

```diff
@@ -1,35 +1,21 @@
-const {
-  SORT_DIRECTIONS,
-  validateSortDirection
-} = require('./sort-pull-requests')
+const { schema } = require('./schema')
+const { DEFAULT_CONFIG } = require('./default-config')
 const { validateReplacers } = require('./template')
 
 const CONFIG_NAME = 'release-drafter.yml'
 
-const DEFAULT_CONFIG = Object.freeze({
-  'change-template': `* $TITLE (#$NUMBER) @$AUTHOR`,
-  'no-changes-template': `* No changes`,
-  'version-template': `$MAJOR.$MINOR.$PATCH`,
-  categories: [],
-  'exclude-labels': [],
-  replacers: [],
-  'sort-direction': SORT_DIRECTIONS.descending,
-  prerelease: false
-})
-
-module.exports.DEFAULT_CONFIG = DEFAULT_CONFIG
-
 module.exports.getConfig = async function getConfig({
   app,
   context,
   getConfig: fetchConfig
 }) {
-  const defaults = {
-    ...DEFAULT_CONFIG,
-    branches: context.payload.repository.default_branch
-  }
+  const repoConfig = await fetchConfig(context, CONFIG_NAME, DEFAULT_CONFIG)
 
-  const config = await fetchConfig(context, CONFIG_NAME, defaults)
+  schema(context).validate(repoConfig)
+
+  const { error, value } = schema(context).validate(repoConfig)
+  const config = value
+  if (error) throw error
```

Comment:
Can we extract this to a function (`validateSchema`) that returns the config or throws?